content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
 
 # Organizations
 
-Organizations are a shared space for [teams][] to collaborate on workspaces in Terraform Cloud.
+Organizations are a shared space for one or more [teams][] to collaborate on workspaces.
 
--> **API:** See the [Organizations API](/cloud-docs/api-docs/organizations). <br/>
-**Terraform:** See the `tfe` provider's [`tfe_organization`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/organization) resource.
+## API and Terraform Enterprise Provider
+
+In addition to the Terraform Cloud UI, you can use the following methods to manage organizations:
+- [Organizations API](/cloud-docs/api-docs/organizations)
+- The `tfe` provider [`tfe_organization`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/organization) resource
 
 ## Selecting Organizations
 
-On most pages within Terraform Cloud, the top navigation bar displays the name of the selected organization. Clicking the name reveals the organization switcher menu, which lists all of the organizations you belong to. You can switch to another organization by clicking its name, or you can create a new organization with the "Create new organization" button.
+Terraform Cloud displays your current organization in the left of the top navigation bar. To select an organization:
 
-![screenshot: the organization switcher menu](/img/docs/org-nav.png)
+1. Click the current organization name to view a list of all the organizations where you are a member.
+1. Click an organization to select it. Terraform Cloud displays list of workspaces within that organization.
 
 ## Joining and Leaving Organizations
 
-To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. See [Organization Settings: Users](#users) below for details.
+To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
```

Comment:
```suggestion
To join an organization, you must be invited by one of its owners and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
```
Not sure if those brackets were meant to be included or not.

Reply:
same as below! they are supposed to be here, but this is an unusual pattern, so I appreciate you calling it out!